elixir/apps/api/test/api/client/channel_test.exs

```diff
@@ -7,7 +7,7 @@ defmodule API.Client.ChannelTest do
 
     Fixtures.Config.upsert_configuration(
       account: account,
-      clients_upstream_dns: [%{address: "1.1.1.1"}]
+      clients_upstream_dns: [%{type: "ip", address: "1.1.1.1"}]
```

Comment:
We should make sure that we add a default port here:
```suggestion
      clients_upstream_dns: [%{type: "ip", address: "1.1.1.1:53"}]
```

Reply:
Rather than add the port to the existing address, I decided to add a second DNS record all together, that way we would have one with a port given and one without.  If you see any issues with that, please let me know.